packages/react-aria-components/src/Table.tsx

```diff
@@ -1213,23 +1244,28 @@ export const Cell = /*#__PURE__*/ createLeafComponent('cell', (props: CellProps,
       isFocused,
       isFocusVisible,
       isPressed,
-      isHovered
+      isHovered,
+      isEditing,
+      isReadOnly
     }
   });
 
   let TD = useElementType('td');
+  let groupState = useMemo(() => ({isReadOnly}), [isReadOnly]);
 
   return (
     <TD
       {...mergeProps(filterDOMProps(props as any), gridCellProps, focusProps, hoverProps)}
       {...renderProps}
       ref={ref}
+      data-editing={isEditing || undefined}
       data-focused={isFocused || undefined}
```

Comment:
Is there a reason for `data-hovered` missing here?

Reply:
Looks like something we missed, I recall any reason why not.